README.md

```diff
@@ -1,31 +1,38 @@
 # Mollie Components Examples
 
-This repository contains examples on how to implement [Mollie Components](https://docs.mollie.com/guides/mollie-components/overview).
+This repository contains examples on how to implement [Mollie Components](https://docs.mollie.com/guides/mollie-components/overview). Mollie Components is a set of Javascript APIs that allows you to add the fields needed for credit card holder data to your own checkout, in a way that is fully PCI-DSS SAQ-A compliant.
 
 ## Documentation
 
-* [Overview](https://docs.mollie.com/guides/mollie-components/overview)
-* [API reference](https://docs.mollie.com/guides/mollie-components/reference)
-* [Handling errors](https://docs.mollie.com/guides/mollie-components/handling-errors) 
-* [Styling](https://docs.mollie.com/guides/mollie-components/styling) 
+- [Integration guide](https://docs.mollie.com/guides/mollie-components/overview)
+- [API reference](https://docs.mollie.com/guides/mollie-components/reference)
+- [Handling errors](https://docs.mollie.com/guides/mollie-components/handling-errors)
+- [Styling](https://docs.mollie.com/guides/mollie-components/styling)
 
-## Example 1 (regular labels)
-[HTML](example-1/index.html) | [CSS](example-1/style.css) | [Javascript](example-1/script.js)
+## Examples
 
-![](screenshots/example-1.gif)
+- [Example 1](./example-1) Bare minimum example ([codepen](example-1))
+- [Example 2](./example-2) Floating labels ([codepen](example-2))
+- [Example 3](./example-3) Compact version ([codepen](example-3))

```

Comment:
should we have a ticket that makes sure that in the end this is edited to the correct links and the amount of actual examples in this repository?

Reply:
yes, you're right. It should be one of the acceptance criteria of the examples